app/assets/stylesheets/home.scss

```diff
@@ -74,7 +74,7 @@ html, body {
 }
 
 #featured {
-  background-color: #1abc9c;
+  background-color: #009688;
```

Comment:
Color literals like `#009688` should only be used in variable declarations; they should be referred to via variable everywhere else.